release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -454,6 +454,46 @@ For more information, see the following sections:
 [id="ocp-4-15-storage"]
 === Storage
 
+[id="ocp-4.15-storage-recovering-vgs-from-prev-installation"]
+==== Recovering volume groups from the previous {lvms} installation
+With this release, the `LVMCluster` custom resource (CR) provides support for recovering volume groups from the previous {lvms} installation. If the `deviceClasses.name` field is set to the name of a volume group from the previous {lvms} installation, {lvms} recreates the resources related to that volume group in the current {lvms} installation. This simplifies the process of using devices from the previous {lvms} installation through the re-installation of {lvms}.
+
+For more information, see xref:../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-creating-lvms-cluster_logical-volume-manager-storage[Creating an `LVMCluster` custom resource (CR)].
+
+[id="ocp-4.15-storage-support-for-wiping-the-devices"]
+==== Support for wiping the devices in {lvms}
+
+This feature provides a new optional field `forceWipeDevicesAndDestroyAllData` in the `LVMCluster` custom resource (CR) to force wipe the selected devices. Prior to this release, wiping the devices required you to manually access the host. With this release, you can force wipe the disks without manual intervention. This simplifies the process of wiping the disks.
```

Comment:
Not critical for this PR, but in the future, recommend avoiding saying "This feature" and instead focusing on what the reader can do.

Reply:
ACK!